### PR TITLE
Add support for Transducer Direct and update Popsicle

### PR DESCRIPTION
**Pull request by mike1808**
* Support pressure transducer from Transducer Direct
* Update Popsicle UUIDs and pressure parsing

### CODE REVIEW

**Comment by mike1808** [comment]
@graphefruit I haven't tested iOS and there are differences in how iOS returns the advertising data.

File: src/classes/devices/common/util.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    return new Uint8Array(adv.kCBAdvDataManufacturerData);
+  }
+
+  if (adv instanceof ArrayBuffer) {
+    const view = new Uint8Array(adv);
+    if (!view.length) {
+      return null;
+    }
+
+    for (let i = 0; i < view.length - 1; ) {
```
Comment (graphefruit) [inline src/classes/devices/common/util.ts:34]: Is this realy working?

Comment (mike1808) [reply src/classes/devices/common/util.ts:34]: What do you mean? Yes, i tested it on Android and it works.

Comment (mike1808) [reply src/classes/devices/common/util.ts:34]: I can add some tests for it

Comment (graphefruit) [reply src/classes/devices/common/util.ts:34]: Alright. The code looked like something missing.
Specially the I++ made me curious

Comment (graphefruit) [reply src/classes/devices/common/util.ts:34]: Therefore good to go. I gonna merge

**Pull request merged**